code/modules/integrated_electronics/passive/power.dm

```diff
@@ -97,11 +97,19 @@
 	activators = list()
 	spawn_flags = IC_SPAWN_DEFAULT|IC_SPAWN_RESEARCH
 	var/volume = 60
-	var/list/fuel = list("plasma" = 10000, "welding_fuel" = 3000, "carbon" = 2000, "ethanol" = 2000, "nutriment" = 1600, "blood" = 1000)
+	var/list/fuel = list("plasma" = 50000, "welding_fuel" = 15000, "carbon" = 10000, "ethanol" = 10000, "nutriment" = 8000)
+	var/multi = 1
+	var/lfwb
 
 /obj/item/integrated_circuit/passive/power/chemical_cell/New()
 	..()
 	create_reagents(volume)
+	var/p = text2path("/datum/config_entry/flag/disable_human_mood")
```

Comment:
What are you even trying to do here ? Why is this path a string and why are you not using the common config helpers?

Reply:
Because there is no such flag in game yet.So it is error sae version of flag check.